### PR TITLE
Dockerized the app

### PR DESCRIPTION
**Pull request by amalic**
build with: docker build -t lodestar .
run with: docker run -d -p 8080:8080 lodestar

use custom config file via volume (-v) switch

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
+FROM freedomkk/tomcat-maven:8
+
+ENV BUILD_DIR /tmp/lodestar/
+
+RUN mkdir $BUILD_DIR && \
+  cd $BUILD_DIR && \
+  git clone https://github.com/amalic/lodestar.git
```
Comment (simonjupp) [inline Dockerfile]: can it use https://github.com/EBISPOT/lodestar by default?

File: web-ui/src/main/resources/lode.properties [hunk 2]
```diff
@@ -1,4 +1,4 @@
-lode.sparqlendpoint.url      http://dbpedia.org/sparql
```
Comment (simonjupp) [inline web-ui/src/main/resources/lode.properties]: can we keep the default to dbpedia?

File: config-docker/lode.properties [hunk 3]
```diff
@@ -0,0 +1,21 @@
+lode.sparqlendpoint.url      http://openlifedata.org/sparql
```
Comment (simonjupp) [inline config-docker/lode.properties]: keep normal defaults? i.e. dbpedia.

**Comment by copitz** [comment]
Hi, just saw this and wanted to add, that I did a docker image a while ago also: https://hub.docker.com/r/netresearch/lodestar/

Sorry for not posting that here earlier - and thanks for your great work!

File: Dockerfile (hunk 1 above)
Comment (amalic) [reply Dockerfile]: yes

File: config-docker/lode.properties (hunk 3 above)
Comment (amalic) [reply config-docker/lode.properties]: yes

File: web-ui/src/main/resources/lode.properties (hunk 2 above)
Comment (amalic) [reply web-ui/src/main/resources/lode.properties]: yes

**Pull request merged**